<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>progressForm</class>
 <widget class="QWidget" name="progressForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="progress_text">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>270</y>
     <width>385</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Connecting ...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="progress_bar">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>320</y>
     <width>385</width>
     <height>27</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="format">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="progress_username">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>20</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="progress_end">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>48</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
